docs/modules/user-guide/partials/ref_devfile-samples.adoc

```diff
@@ -3,5 +3,6 @@
 
 .Additional resources
 
-* link:https://github.com/devfile/api/issues[Creating devfiles].
-* link:https://github.com/devfile/docs[Documenting devfiles].
+To build and test devfiles, visit our link:https://github.com/devfile/api[GitHub API repository]. Here, you can also view open issues, so you can see the creation of devfiles.
+
+To build and contribute to the documentation, visit our link:https://github.com/devfile/docs[GitHub Docs repository].
```

Comment:
Maybe "our GitHub Docs repository" is some standard, but it sounds strange to me. Shouldn't it be "your" or "Red Hat's", depending on what you are talking about?

Reply:
it's of course possible that I don't know what I am talking about and this makes perfect sense there.

Reply:
Makes perfect sense, Eva :) Resolving as I have addressed this valid concern.